auto_round/script/llm.py

```diff
@@ -485,11 +485,16 @@ def tune(args):
     layer_config = {}
     not_quantize_layer_names = get_fp_layer_names(model, args.fp_layers)
     for name in not_quantize_layer_names:
-        layer_config[name] = {"bits": 16, "act_bits": 16}
+        layer_config[name] = {"bits": 16, "act_bits": 16, "data_type": "float", "act_data_type": "float"}
```

Comment:
why this is required?  there should be same issue in API side, we'd better to fix it in auto_round.py code

Reply:
if not of the dytpe matched, raise the error

Reply:
will fix API config on other PR